loader/icd.h

```diff
@@ -152,40 +154,40 @@ void khrIcdContextPropertiesGetPlatform(
     cl_platform_id *outPlatform);
 
 // internal tracing macros
-#if 0
-    #include <stdio.h>
-    #define KHR_ICD_TRACE(...) \
+#define KHR_ICD_TRACE(...) \
+if (khrEnableTrace) \
```

Comment:
This new if check should be moved inside of the one-trip do-while loop, rather than be placed outside.

For details, see: https://wiki.sei.cmu.edu/confluence/display/c/PRE10-C.+Wrap+multistatement+macros+in+a+do-while+loop

Applies here and the other places in this file where an if check was added.